code/style.css

```diff
@@ -1,4 +1,253 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background-color: #002233;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  padding: 20px;
 }
+
+
+/*** MOBILE MAIN DESIGN ***/
+
+@media (min-width: 1px) and (max-width: 667px) {
```

Comment:
When you're coding mobile first, you don't have to have a media query for mobile :) You can just write all the mobile styling as the default styling, then have media queries for tablet and desktop